### DISCUSSION TITLE
Does pixi3D have any support for animation blending?

### DISCUSSION THREAD

**Discussion by Unwarped** [jnsmalm/pixi3d#194] [Q&A] [answered]
Good to see that you are still working on this jnsmalm.
I was wondering does pixi3D support blending between two animations? Like going from idle to walking?
Thanks!

**Comment by jnsmalm** [comment, marked as answer]
Unfortunately not at this time.

**Comment by Unwarped** [comment]
Thanks jnsmalm. Happy new year!

Do you have any plans to update Pixi3D for PixiJS V8?

**Reply by jnsmalm** [reply]
Hey, happy new year! I would like to add support for PixiJS 8, but at this time I have to work on so much other stuff so it's hard to find the time.
